src/main/java/nl/tudelft/scrumbledore/GUI.java

```diff
@@ -366,6 +372,9 @@ private void advanceLevel() {
    */
   private void checkPlayerAlive() {
     if (!game.getCurrentLevel().getPlayer().isAlive()) {
+      // Logging that the game has been restarted.
+      Logger.log("--------------------PLAYER RESTARTED THE GAME--------------------");
```

Comment:
The reason there are two events with 20 lines surrounding the log is that these are events that control the game flow.  For example, restarting the game, pausing the game, terminating the game, etc.  Useful for future implementation of, for example, a replay mode.